### PR TITLE
Add verbose option to writeManifest()

### PR DESCRIPTION
**Pull request by andrie**
Add `verbose = FALSE` argument to writeManifest(), and pass this through to createAppManifest(), in order to benefit from improved verbose logging in packrat.

### CODE REVIEW

File: R/bundle.R [hunk 1]
```diff
@@ -937,13 +957,15 @@ performPackratSnapshot <- function(bundleDir) {
   }
 
   # generate a snapshot
-  suppressMessages(
+  # suppressMessages(
```
Comment (jmcphers) [inline R/bundle.R:960]: Why is this commented out? Do we want to see these messages even if `verbose` is turned off?

File: R/bundle.R [hunk 2]
```diff
@@ -853,6 +869,7 @@ addPackratSnapshot <- function(bundleDir, implicit_dependencies = c()) {
     # rethrow error so we still halt deployment
     stop(e)
   })
+  # if (verbose) logger(" - done performing packrat snapshot")
```
Comment (jmcphers) [inline R/bundle.R:872]: Here and elsewhere -- it is not clear why this code is commented out; please either uncomment it or remove it.

**Review by jmcphers — CHANGES_REQUESTED** [review]

File: R/bundle.R (hunk 1 above)
Comment (andrie) [reply R/bundle.R:1003]: Fixed in latest commit

File: R/bundle.R (hunk 2 above)
Comment (andrie) [reply R/bundle.R]: Fixed in latest commit

**Review by jmcphers — APPROVED** [review]

**Pull request merged**